drivers/pwm/Kconfig.rpi_pico

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2022, Joep Buruma
+# SPDX-License-Identifier: Apache-2.0
+
+config PWM_RPI_PICO
+	bool "RPi Pico PWM"
+		default y
+		depends on SOC_SERIES_RP2XXX
+	select PICOSDK_USE_PWM
```

Comment:
please fix formatting here, and default based on dt_compat_status (check some other drivers)